cmd/create/cluster/cmd.go

```diff
@@ -3017,6 +3077,55 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 }
 
+func GetBillingAccountContracts(cloudAccounts []*accountsv1.CloudAccount,
+	billingAccount string) ([]*accountsv1.Contract, bool) {
+	var contracts []*accountsv1.Contract
+	var isContractEnabled bool
+	for _, account := range cloudAccounts {
+		if account.CloudAccountID() == billingAccount {
+			contracts = account.Contracts()
+			for _, contract := range contracts {
+				for _, dimension := range contract.Dimensions() {
+					if ocm.HasValidContracts(dimension) {
+						isContractEnabled = true
+						break
+					}
+				}
+			}
+
+		}
+	}
+	return contracts, isContractEnabled
+}
+
+func PrintContracts(contracts []*accountsv1.Contract, r *rosa.Runtime) {
+	for _, contract := range contracts {
+		format := "Jan 02, 2006"
+		dimensions := contract.Dimensions()
+		var prePurchaseInfo string
+
+		for _, dimension := range dimensions {
+			numberOfVPCs := ocm.GetDimensionValue(dimension, "four_vcpu_hour")
+			numberOfClusters := ocm.GetDimensionValue(dimension, "control_plane")
+			prePurchaseInfo = prePurchaseInfo + fmt.Sprintf("   | Number of vCPUs:    |'%s'             | \n"+
+				"   | Number of clusters: |'%s'             | \n",
+				strconv.Itoa(numberOfVPCs), strconv.Itoa(numberOfClusters))
+
+		}
+
+		r.Reporter.Infof(
+			"\n"+
+				"   Contract enabled for this billing account. \n"+
```

Comment:
Looking at the design doc, it seems "Using [account number ] as billing account" is missing.

It should go before the "Contract enabled for this billing account" text, PTAL.

Reply:
It will always show up if the billing account is not the AWS infrastructure account.
<img width="1362" alt="image" src="https://github.com/openshift/rosa/assets/26282541/bc5c8025-2ec3-4d35-aec0-182d7a4002d3">